pennylane/ops/op_math/condition.py

```diff
@@ -279,26 +273,22 @@ def __call_capture_disabled(self, *args, **kwargs):
         for pred, branch_fn in zip(self.preds, self.branch_fns):
             if pred:
                 return branch_fn(*args, **kwargs)
-
-        return self.false_fn(*args, **kwargs)
+        if self.otherwise_fn:
```

Comment:
Why do we have two names `otherwise_fn` and `false_fn` of the same thing?

Reply:
🤷 This has caused me a lot of confusion too.